### PR TITLE
default deltaT is a little bigger

### PR DESCRIPTION
**Pull request by shaomeng**
This PR increases the default deltaT by a factor of 3.

### CODE REVIEW

**Comment by shaomeng** [comment]
Well, there's unexpected behavior... I'll need to do more work here...

**Comment by shaomeng** [comment]
@clyne  This PR is ready for merge now.

File: lib/flow/VaporField.cpp [hunk 1]
```diff
@@ -436,21 +436,25 @@ int VaporField::CalcDeltaTFromCurrentTimeStep( float& delT ) const
             }
 
     // Let's find the maximum velocity on these sampled locations
+    // Note that we want the raw velocity, which will be the value
+    // returned by GetVelocity() divided by the velocity multiplier.
+    const float mult1o = 1.0f / float(_params->GetVelocityMultiplier());
```
Comment (clyne) [inline lib/flow/VaporField.cpp]: Need to  avoid divide by zero

Comment (shaomeng) [reply lib/flow/VaporField.cpp]: It cannot be zero. FlowParams defaults it to be 1.0, and FlowSubtabs rejects user input that is zero. 
That said, I've added an extra layer of guard in commit cb8e8b6

Comment (clyne) [reply lib/flow/VaporField.cpp]: You mean it *should not* be zero :-). What happens if our future python or web interface sets it to zero? It just good coding practice to not rely on what is supposed to happen somewhere else in the code.

**Pull request merged**